java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/AzureEventHubsSourceTask.java

```diff
@@ -28,29 +45,51 @@ public String version() {
 
   @Override
   public void start(Map<String, String> props) {
-    log.info("Task initialising");
-    printAsciiHeader(jarManifest, "/azure-eventhubs-ascii.txt");
+    Optional<SourceTaskContext> contextOptional = ofNullable(context);
+    Map<String, String> configProps = null;
+    if (contextOptional.isPresent()) {
```

Comment:
It's down to my lack of knowledge maybe, but why is the `context` here used to get the properties and ignore the `props` parameter provided here?